src/components/NonProfit/NonProfit.js

```diff
@@ -111,6 +115,17 @@ export default function NonProfit(props) {
     } else {
       if (nonprofit?.problem_statements?.length === 0) {
         return <div>Working on it!</div>;
+      } else if (!nonprofit?.problem_statements) {
+        return (
+          <Grid container justifyContent=''>
+            <Grid item style={{ fontSize: "13px"}} xs={12}>
+              Looks like there are no active projects with this organization. Let us know how we can help:<br/>
+            </Grid>
+            <ApplyButtonContainer>
+              <ApplyButton href="https://www.ohack.dev/nonprofits/apply">Submit your project ideas!</ApplyButton>
```

Comment:
No need to use the full URL here, just `/nonprofits/apply` - this will actually take people to the page much faster than when you use the full URL since it has to establish a new connection - you also don't want to hardcode a URL in case it ever changes.